.gitignore

```diff
@@ -6,6 +6,7 @@ coverage.html
 
 # Common in OSs and IDEs
 .idea
+.vscode
```

Comment:
Happy to include this in karpenter, but I also recommend using a .gitignore.global in your ~/.gitconfig 
```
	excludesfile = /Users/etarn/.gitignore.global
```
This avoids needing to add all of these env specific files to every project's gitignore.